pkg/auth/config/secret.go

```diff
@@ -98,15 +117,16 @@ var _ = SecretConfigSchema.Add("SecretKey", `{ "type": "string" }`)
 type SecretKey string
 
 const (
-	DatabaseCredentialsKey   SecretKey = "db"
-	RedisCredentialsKey      SecretKey = "redis"
-	SMTPServerCredentialsKey SecretKey = "mail.smtp"
-	TwilioCredentialsKey     SecretKey = "sms.twilio"
-	NexmoCredentialsKey      SecretKey = "sms.nexmo"
-	JWTKeyMaterialsKey       SecretKey = "jwt"
-	OIDCKeyMaterialsKey      SecretKey = "oidc"
-	CSRFKeyMaterialsKey      SecretKey = "csrf"
-	WebhookKeyMaterialsKey   SecretKey = "webhook"
+	DatabaseCredentialsKey    SecretKey = "db"
+	RedisCredentialsKey       SecretKey = "redis"
+	OAuthClientCredentialsKey SecretKey = "oauth.client.external"
```

Comment:
I suggest to have a 'sso' prefix to avoid confusing our clients (as IdP) and us as a client (as RP).

Reply:
I added the `external` suffix for this purpose. But I agree the sso prefix makes more sense. Let me change it.